<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevicePage</class>
 <widget class="QWidget" name="DevicePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>575</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {
	min-height: 40px;
	min-width: 5px;
	font-size: 12px;
	font-weight: 300;
}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="PerformanceTab">
    <attribute name="title">
     <string>Performance</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="LightingTab">
    <attribute name="title">
     <string>Lighting</string>
    </attribute>
    <widget class="QComboBox" name="LightingTypeDropdown">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="KeyTab">
    <attribute name="title">
     <string>Keys</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="KeyStatsTab">
    <attribute name="title">
     <string>Key Stats</string>
    </attribute>
    <widget class="QPushButton" name="StatsStartButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StatsStopButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="StatsLeftClickLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Left clicks:</string>
     </property>
    </widget>
    <widget class="QLabel" name="StatsRightClickLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Right clicks:</string>
     </property>
    </widget>
    <widget class="QLabel" name="StatsDistanceLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance:</string>
     </property>
    </widget>
    <widget class="QLabel" name="StatsDistanceText">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0 cm</string>
     </property>
    </widget>
    <widget class="QLabel" name="StatsRightClickText">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="StatsLeftClickText">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="BatteryTab">
    <attribute name="title">
     <string>Battery</string>
    </attribute>
    <widget class="QLabel" name="BatteryChargeText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 15px;
	font-weight: 500;
}</string>
     </property>
     <property name="text">
      <string>N/A%</string>
     </property>
    </widget>
    <widget class="QComboBox" name="TimeToSleepDropdown">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>17</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Never</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1min</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2min</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3min</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5min</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10min</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Time to sleep:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Warning at:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="WarningAtInput">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>47</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="WarningAtNeverButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>83</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Never</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="CalibrationTab">
    <attribute name="title">
     <string>Calibration</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="ProfileTab">
    <attribute name="title">
     <string>Profile</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="ProfileGrid"/>
    </widget>
   </widget>
   <widget class="QWidget" name="InformationTab">
    <attribute name="title">
     <string>Information</string>
    </attribute>
    <widget class="QLabel" name="VersionLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Version:</string>
     </property>
    </widget>
    <widget class="QLabel" name="DongleVersionLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2.4GHz Dongle:</string>
     </property>
    </widget>
    <widget class="QLabel" name="VersionText">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Not available</string>
     </property>
    </widget>
    <widget class="QLabel" name="DongleVersionText">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Not available</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ResetCurrentProfileButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset Current Profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ResetAllProfilesButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset All Profiles</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="SaveProfileButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>575</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Profile</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
